CHANGELOG.md

```diff
@@ -22,6 +22,7 @@ The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/).
 - Vi mode keybinding (z) to center view around vi mode cursor
 - Accept hexadecimal values starting with `0x` for `--embed`
 - Config option `cursor.blink_timeout` to timeout cursor blinking after inactivity
+- On macOS, Alacritty now reads `AppleFontSmoothing` from user defaults to control font smoothing
```

Comment:
Could you move it to `CHANGED`?

Reply:
I could, but I put it in `Added` in `crossfont`. I think it makes more sense as an `Added` and a `Removed`. The only `Changed` in `crossfont` was a rename.

Reply:
For alacritty it's more of a `CHANGED`, since it was using thin strokes before, but via config option. Now you'll have to use apple user default.

Though. I'm fine with keeping it in added.

Reply:
I'll leave it as-is. The default for alacritty was `false`, contra the comment in the `alacritty.yaml`, and it's been not working for two years anyway. Might as well be "added" for most users. ;)

Reply:
This should be moved to `CHANGED`.

Reply:
Whoops, didn't see this one since the conversation was still resolved. Moved.